e2e_tests/test_homepage.py

```diff
@@ -3,7 +3,7 @@
 
 
 def test_has_title(page: Page):
-    page.goto("http://localhost:5000/")
+    page.goto("/")
```

Comment:
We can configure the base url in a config file, in the test files or in the cli commands